source/dshowcapture.cpp

```diff
@@ -83,6 +85,14 @@ bool Device::SetAudioConfig(AudioConfig *config)
 	return context->SetAudioConfig(config);
 }
 
+void Device::SaveSettings(const std::string& filePath) {
+    SaveSettingsToFile(filePath + "device-settings.json");
+}
+
+void Device::LoadSettings(const std::string& filePath) {
+    LoadSettingsFromFile(filePath + "device-settings.json");
```

Comment:
I would put the file name `device-settings` in a define / constant variable and then re use it instead of copy pasting the name in multiple places of the code.

Reply:
Gong to make the change

Reply:
Change done

Reply:
Perfect 👍